chapter_11_cqrs.asciidoc

```diff
@@ -103,6 +118,20 @@ def test_unhappy_path_returns_400_and_error_message():
 ----
 ====
 
+///
+// (ej) The API routes are a bit un-conventional. For example, instead of 
+//      * POST /allocate then GET /allocations/<order_id>
+//      something like this is more typical:
+//      * POST /allocations then GET /allocations?order_id=<order_id>
+//
+//      The second style states the API intentions directly (POST creates allocation
+//      sub-resources, then GET is just a query over all allocations)  With the first style
+//      I have to make guesses at the the API's intentions and dig into the implementaion 
+//      to be sure, and it seems odd that an order_id is a sub-resource of an allocation.
+//
+//      This would make the examples a easier to follow, and adhere to
+//      a RESTful API more closely.
```

Comment:
you know, i've never actually built an API that follows this pattern -- another django hangover probably -- but i totally agree with the reasoning.  will put it on the list.